### DISCUSSION TITLE
ObserversRpc with [IncludeOwner = false]

### DISCUSSION THREAD

**Discussion by Xelopie** [FirstGearGames/FishNet#245] [Feature Request]
I have just noticed that ObserversRpc with [IncludeOwner = false] doesn't mean that the call message will not be sent to the owner. Instead, the RPC call is sent to the owner but the owner will simply ignore it. 

So my point is, why don't just skip the sending to the owner on server side when using ObserversRpc with IncludeOwner = false? This can help saving some bandwidth especially when using client authoritative movement.

Ofc I can do it like 
```
foreach (var observer in Observers) {
  if (observer == Owner) return;
  // Call Rpc
  xxxRpc();
}
```
But this means more overhead than handling this internally, right?

**Comment by FirstGearGames** [comment]
There were reasons it was done this way in the past but they are no longer required. I'll get this improvement added in.

**Comment by FirstGearGames** [comment]
Added in 3.3.0 but renamed to match other API. See https://fish-networking.gitbook.io/docs/manual/general/changelog/break-solutions#3.3.0

**Reply by Xelopie** [reply]
Thanks and noted!
